src/ioctls/vcpu.rs

```diff
@@ -1254,6 +1325,17 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_set_signal_mask() {
+        use libc::*;
+        use vmm_sys_util::signal;
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let vcpu = vm.create_vcpu(0).unwrap();
+        let sigset = signal::create_sigset(&[SIGINT, SIGKILL, SIGTERM]).unwrap();
+        assert!(vcpu.set_signal_mask(&sigset).is_ok());
```

Comment:
I'd prefer the test to prove that the functionality works, not just cover the lines.

Reply:
That's a good point! Thank you.
But as the later comments mentioned, it's quite probable that this PR will be canceled.
I will improve this if this PR continue.